src/keycloak/chart/values.yaml

```diff
@@ -371,3 +371,13 @@ thirdPartyIntegration:
     # The Client Certificate format used by the L7 Load Balancer.
     # Allowed settings: "PEM" and "AWS"
     tlsCertificateFormat: PEM
+
+# Extra volumes to mount in the StatefulSet
+extraVolumes: []
+
+# Extra volume mounts to mount in the Keycloak container
+extraVolumeMounts: []
```

Comment:
I don't have any immediate concerns here but just to ask - are there any issues with exposing volumes generically here? Or would we want to restrict the directory or something to tailor this for tmp dirs, etc only?

It seems like a pretty generic function to include in a helm chart but I know we're generally cautious around exposing too many knobs (like the args) to make sure we don't have easy security holes.

Reply:
I generally agree - wasn't sure if there might be directories we'd want to limit due to Keycloak automatically loading content from there (such as [the ones we leverage for identity-config](https://github.com/defenseunicorns/uds-identity-config/blob/main/src/sync.sh#L9-L13)). Overall I think this is fine and probably enables more interesting config beyond the initial intent - just wanted to drop a note in case @slaskawi or someone else could think of an issue with this.

Ultimately we want UDS to be "more secure" but always a balance in security and configurability - ensuring we have a good secure baseline but also generally exposing _some knobs_ for more flexibility.

Reply:
If you're configuring Keycloak, your're a cluster admin. So I'm not too worried security-wise about someone using arbitrary volumes. Assuming it isn't too error prone or something.

And I think all the DoD certs can exceed the size of a single ConfigMap if I remember. So mounting two here is nice

Reply:
> And I think all the DoD certs can exceed the size of a single ConfigMap if I remember. So mounting two here is nice

Tangent but so far we've been okay putting all the CAs into a single secret (we do this for the gateway CA). Looks like maybe [credentialNames](https://istio.io/latest/docs/reference/config/networking/gateway/#ServerTLSSettings-credential_names) could be an option once we hit that threshold (I think we're close).